File: Examples/VisualBasic/AsposePdfFacades/Images/ReplaceImage.vb
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades
Namespace AsposePDFFacades.Images
    Public Class ReplaceImage
        Public Shared Sub Run()
            ' ExStart:ReplaceImage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Images()
            ' Open input PDF
            Dim pdfContentEditor As New PdfContentEditor()
            pdfContentEditor.BindPdf(dataDir & Convert.ToString("ReplaceImage.pdf"))
            ' Replace image on a particular page
            pdfContentEditor.ReplaceImage(1, 1, dataDir & Convert.ToString("aspose-logo.jpg"))
            ' Save output PDF
            pdfContentEditor.Save(dataDir & Convert.ToString("ReplaceImage_out_.pdf"))
            ' ExEnd:ReplaceImage            
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfFacades/Images/ReplaceImage.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace Aspose.Pdf.Examples.CSharp.AsposePDFFacades.Images
{
    public class ReplaceImage
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Images();
            //open input PDF
            PdfContentEditor pdfContentEditor = new PdfContentEditor();
            pdfContentEditor.BindPdf(dataDir+ "ReplaceImage.pdf");
            //replace image on a particular page
            pdfContentEditor.ReplaceImage(1, 1, dataDir+ "aspose-logo.jpg");
            //save output PDF
            pdfContentEditor.Save(dataDir+ "ReplaceImage_out.pdf");
            
            
        }
    }
}